### DISCUSSION TITLE
Error: connect ECONNREFUSED 2600:1f1c:f9:4d08:e10b:2f2:c1fa:2566:5432

### DISCUSSION THREAD

**Discussion by palodaman** [supabase/supabase#29580] [Questions]
trying to follow this guide a setup a supabase project with an expo tech stack[ ](https://supabase.com/docs/guides/getting-started/tutorials/with-expo-react-native) but when i try to setup the database schema using the user management starter, and click on run, it gives me the following error: 
Error: connect ECONNREFUSED 2600:1f1c:f9:4d08:e10b:2f2:c1fa:2566:5432
Try refreshing your browser, but if the issue persists, please reach out to us via support.

When i try to see the database to see if it has been set up, it loads for a while, and then says the following:
Failed to retrieve tables
Error: failed to get pg.tables: connect ECONNREFUSED 2600:1f1c:f9:4d08:e10b:2f2:c1fa:2566:5432

I have also tried inviiting a team member to the project and they get the same error. Could someone please explain why that could be?

PS: when i checked the project status, it says to unable to connect under the database tab

**Comment by GaryAustin1** [comment]
Please contact support.

You can try and restart your instance.

Sounds like normal different browser/device won't help if you have a team member not accessing.
